plugins/com.google.cloud.tools.eclipse.util/src/com/google/cloud/tools/eclipse/util/jobs/PluggableJob.java

```diff
@@ -17,9 +17,8 @@
 package com.google.cloud.tools.eclipse.util.jobs;
 
 import com.google.common.base.Preconditions;
-import com.google.common.base.Predicate;
-import com.google.common.base.Predicates;
 import java.util.concurrent.Callable;
+import java.util.function.Predicate;
```

Comment:
Weird. Why do we import this if we don't use it?

Reply:
`stalenessCheck` on [line 32](https://github.com/GoogleCloudPlatform/google-cloud-eclipse/pull/2826/files#diff-5de80434c98534b2471cb48f5a26d8e3R32)